src/components/ChatMenu.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import IconButton from 'material-ui/IconButton';
+import Menu, { MenuItem } from 'material-ui/Menu';
+import MoreVertIcon from 'material-ui-icons/MoreVert';
+
+class LongMenu extends React.Component {
+  state = {
+    anchorEl: null
+  };
+
+  handleClick = event => {
+    this.setState({ anchorEl: event.currentTarget });
+  };
+
+  handleClose = () => {
+    this.setState({ anchorEl: null });
+  };
+
+  render() {
+    const { anchorEl } = this.state;
+    const { actions, user, isConnected } = this.props;
+
+    let action = {};
```

Comment:
Can be simplified as:

```
const action = {
  name: user.isCreator ? 'Delete' : 'Leave',
  onClick: user.isCreator ? actions.deleteChat : actions.leaveChat
}
```

But that's a matter of taste.

Reply:
I agree, it looks better